deb/openmediavault-nut/usr/share/openmediavault/datamodels/conf.service.nut.json

```diff
@@ -11,13 +11,34 @@
 			"type": "boolean",
 			"default": false
 		},
+		"mode": {
+			"type": "string",
+			"enum": [ "netclient", "standalone" ],
+			"default": "standalone"
+		},
 		"upsname": {
 			"type": "string",
 			"pattern": "^[a-z0-9_-]+$"
 		},
 		"comment": {
 			"type": "string"
 		},
+		"nutserver": {
+			"type": "string"
+		},
+		"nutserverlogin": {
+			"type": "string",
+			"default": ""
+		},
+		"nutserverpassword": {
```

Comment:
```suggestion
		"remotepassword": {
```

Reply:
`remotepassword` is already taken and used for remote monitor under standalone mode.
Perhaps `netclientpassword` will be better?

https://github.com/openmediavault/openmediavault/blob/811229a7c9e90682df1c17849f1ddcbe0948947a/deb/openmediavault-nut/usr/share/openmediavault/datamodels/conf.service.nut.json#L59-L64